### ISSUE TITLE
regions sorting

### ISSUE THREAD

**Issue by mxdpeep** [pearkes/tugboat#164]
please, sort the regions listing by name :dancer: 

looks dizzy

```
Regions:
New York 1 (id: 1)
San Francisco 1 (id: 3)
New York 2 (id: 4)
Amsterdam 2 (id: 5)
Singapore 1 (id: 6)
London 1 (id: 7)
New York 3 (id: 8)
Amsterdam 3 (id: 9)
```

**Comment by petems** [comment]
This was made back in the ye olde days when there was only 3 regions :smile:

**Comment by petems** [comment]
Fixed by #165:

Example
```ruby
Regions:
Amsterdam 1 (id: 2) (slug: ams1)
Amsterdam 2 (id: 5) (slug: ams2)
Amsterdam 3 (id: 9) (slug: ams3)
Frankfurt 1 (id: 10) (slug: fra1)
London 1 (id: 7) (slug: lon1)
New York 2 (id: 4) (slug: nyc2)
New York 3 (id: 8) (slug: nyc3)
San Francisco 1 (id: 3) (slug: sfo1)
Singapore 1 (id: 6) (slug: sgp1)
```
